docs/user/guides/setup/index.rst

```diff
@@ -28,7 +28,7 @@ The following how-to guides help you solve common tasks and challenges in the se
     Is your version (flyout) menu overwhelmed and hard to navigate?
     Here's how to make it shorter.
 
-⏩️ :doc:`Changing the versioning scheme of your project </guides/setup/versioning-schemes>`
+⏩️ :ref:`versioning-schemes:How to change the URL versioning scheme of your project`
```

Comment:
I think this is the correct Sphinx syntax, but I'm always confused 🙃 

```suggestion
⏩️ :ref:`How to change the URL versioning scheme of your project <versioning-schemes>`
```

Reply:
The link seems to work?